runtime/oti/j9javaaccessflags.h

```diff
@@ -118,6 +119,8 @@
 #define J9AccVolatile 0x40
 #define J9AccRecord 0x400
 #define J9AccSealed 0x200
+#define J9AccHiddenClassOptionNestmate 0x80
```

Comment:
Are you sure these bits are available? I see overlap with varags (likely doesn't apply to classes) and ValueType.

Reply:
> However, I noticed both J9AccVarArgs and J9AccTransient are 0x80, and they are both set to romClass->modifiers.

This comment can be ignored. After a closer look, they are setting to romMethod->modifiers and romField->modifiers.